paths/paths.go

```diff
@@ -3,60 +3,61 @@ package paths
 import (
 	"os"
 	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
 )
 
 // node repo path defaults
 const filPathVar = "FIL_PATH"
-const defaultHomeDir = "~/.filecoin"
-const defaultRepoDir = "repo"
+const defaultRepoDir = "~/.filecoin"
 
 // node sector storage path defaults
 const filSectorPathVar = "FIL_SECTOR_PATH"
-const defaultSectorDir = "sectors"
+const defaultSectorDir = ".filecoin_sectors"
```

Comment:
opinion: underscores are harder to type than minus signs, so I prefer them outside of super-constrained contexts like identifier names in code.